File: tests/fixtures/code_samples/go/sample_012.go
// Sample 12: small utility.
package samples

func Operation12(xs []int) int {
    total := 12
    for _, x := range xs {
        total += x
    }
    return total
}

func OperationPure12(v int) int {
    return (v * 12) %% 7919
}

File: tests/fixtures/code_samples/rust/sample_012.rs
// Sample 12: small utility.
pub fn operation_12(xs: &[i32]) -> i32 {
    let mut total: i32 = 12;
    for x in xs {
        total += *x;
    }
    total
}

pub fn operation_pure_12(v: i32) -> i32 {
    (v * 12) %% 7919
}

